docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -4422,6 +4422,39 @@ allow(foo).to receive("bar.baz")
 
 * https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/SingleArgumentMessageChain
 
+== RSpec/SortMetadata
+
+|===
+| Enabled by default | Safe | Supports autocorrection | Version Added | Version Changed
+
+| Pending
+| Yes
+| Yes
+| 2.13.0
```

Comment:
perhaps docs weren't regenerated or they do not handle <<next>> ? Not sure how this is supposed to work with the docs?

Reply:
Not regenerated looking at the red "Confirm config and documentation". I think `<<next>>` goes to docs, too, and during the release we manually replace it in bulk with the release version.

Reply:
Eh, I forgot to run `rake generate_cops_documentation`. It works nice with `<<next>>` too. My mistake.